app/src/main/java/com/example/parkingsystem/utilities/Picker.java

```diff
@@ -46,6 +45,8 @@ public void onTimeSet(TimePicker view, int hourOfDay, int minute) {
                         calendar.set(HOUR_OF_DAY, hourOfDay);
                         calendar.set(MINUTE, minute);
 
+                        @SuppressLint("SimpleDateFormat")
```

Comment:
you can use `new SimpleDateFormat(SIMPLE_DATE_FORMAT, Locale.getDefault()` to avoid this warning